packages/manager/modules/manager-pci-common/src/components/rclone-download/StorageRegions.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslatedMicroRegions } from '@ovh-ux/manager-react-components';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { OdsSelect, OdsSpinner, OdsText } from '@ovhcloud/ods-components/react';
+import { useStorageRegions } from '../../api/hook/useRegions';
+import { TRegion } from '../../api/data/regions';
+
+interface S3StorageRegionsProps {
+  projectId: string;
+  onStorageRegionChange: (value: string) => void;
+}
+
+export default function StorageRegions({
```

Comment:
This seems a duplicate of S3StorageRegions

Reply:
it looks like it is not strictly the same, the data source is different, one of the component is also displaying a specific text caption

Reply:
Could you factorize these ?